rgsync/redis_gears_write_behind.py

```diff
@@ -426,3 +458,77 @@ def __init__(self, GB, keysPrefix, mappings, connector, name, version=None):
         filter(TryWriteToTarget(self)).\
         foreach(UpdateHash).\
         register(mode='sync', prefix='%s*' % keysPrefix, eventTypes=['hset', 'hmset'], convertToStr=False)
+
+
+class RGJSONWriteBehind(RGWriteBase):
+    # JSONWrite Behind
+    # The big deal is that:
+    # 1. It calls ValidateJSONHash instead of ValidateHash
+    # 2. The init requires a data key, in order to go through the sub map and update
+    #    within the JSON document.
+    def __init__(self, GB, keysPrefix, connector, name, version=None,
+                 onFailedRetryInterval=5, batch=100, duration=100,
+                 eventTypes=['json.set', 'json.del',
+                             'json.strappend', 'json.arrinsert', 'json.arrappend',
+                             'json.arrtrim', 'json.arrpop'],
```

Comment:
`change` event for crdt. Also what about del/rename?

Reply:
Thanks for _change_ I removed it as I didn't have context. Added back, including del. Given that we haven't supported rename elsewhere, I'm advocating for not supporting it here / and haven't.  I including json only commands originally, but now in.